refract-spec.md

```diff
@@ -105,11 +101,193 @@ In addition to expressing Refract Elements as objects with `element`, `attribute
   - (array)
   - (object)
   - (Compact Element)
-  - array[Compact Element]
+  - (array[Compact Element])
+
+##### Example
+
+Below is a Refract element `foo` that is expressed in the normal Refract representation.
+
+```json
+{
+  "element": "foo",
+  "content": "bar"
+}
+```
+
+This is how it would represented in the compact version.
+
+```json
+["foo", {}, {}, "bar"]
```

Comment:
It is OK. It may just be confusing to some, maybe then `["foo", null, null, "bar"]` should be a preferred notation to not hint element or hash.